packages/web/src/components/data-entry/TagInput.js

```diff
@@ -38,6 +38,9 @@ class TagInput extends Component {
     if (this.state.typingMode && this.newTagInputRef.current) {
       this.newTagInputRef.current.focus()
     }
+    if (this.state.tags !== this.props.tags) {
```

Comment:
is the performance okay with this? ive seen controlled components use the prop directly on the element to prevent needing a rerender